app/components/appeals-app-services/clamav.tf

```diff
@@ -15,7 +15,7 @@ resource "azurerm_container_group" "clamav" {
     name   = "clamav"
     image  = "clamav/clamav:latest_base"
     cpu    = "1"
-    memory = "3"
+    memory = "4"

```

Comment:
reckon this might be the issue? thought I had seen a recommendation of 3 at some point

Reply:
They say 3 minimum, 4 recommended:
https://docs.clamav.net/manual/Installing/Docker.html